app/models/account/transaction_builder.rb

```diff
@@ -21,7 +21,7 @@ def transfer?
     end
 
     def create_transfer
-      return create_unlinked_transfer(account.id, signed_amount) unless transfer_account_id
+      return create_unlinked_transfer(account.id, signed_amount) if transfer_account_id.blank?
```

Comment:
in this case the value of `transfer_account_id` is an empty string ""

Reply:
nice catch!